openapi.yaml

```diff
@@ -2993,6 +2995,319 @@ paths:
           $ref: '#/components/responses/501MethodNotImplemented'
     parameters:
       - $ref: '#/components/parameters/workspaceId'
+  '/workspaces/{workspaceId}/keys':
+    post:
+      tags:
+        - Keys
+      x-sdkOperation: create
+      x-isPost: true
+      summary: Create a key
+      description: Creates a Key object.
+      operationId: keyCreate
+      parameters:
+        - $ref: '#/components/parameters/X-Api-Key'
+        - $ref: '#/components/parameters/Key-Accept'
+        - $ref: '#/components/parameters/Key-Content-Type'
+      requestBody:
+        content:
+          application/vnd.whispir.key-v1+json:
```

Comment:
Updated. Instead of using `application/json` directly, I created a reference `Unified-Content-Type` with value `application/json` so that this reference can also be used by new apis.